tests/test_flow/test_persistence.py

```diff
@@ -1069,11 +1069,13 @@ def ref_func():
 def test_versioning_auto_version_options(builder):
     builder.set("core__versioning_mode", "auto")
 
-    complex_number = complex(1.0, 0.0)
+    import pandas as pd
+
+    df = pd.Series([1, 2, 3])
```

Comment:
Doesn't `df` generally mean "dataframe"?

Reply:
You're right. I was creating a dataframe first but ended up with a series. I'll rename it.